applications/matter_bridge/README.rst

```diff
@@ -140,11 +140,12 @@ Getting a list of Bluetooth LE devices available to be added
 
    .. code-block:: console
 
-      I: Scan result:
-      I: -----------------------------
-      I: | Index |      Address      |
-      I: -----------------------------
-      I: | 0     | e6:11:40:96:a0:18 |
+      Scan result:
+      ---------------------------------------------------------------------
+      | Index |      Address      |                   UUID
+      ---------------------------------------------------------------------
+      | 0     | e6:11:40:96:a0:18 | 0x181a (Environmental Sensing Service)
```

Comment:
Does the UUID directly map to a BLE device?
so the (Environmental Sensing Service) is not needed if the `0x181a` map to only that device

Reply:
I'm not sure I understand your question. The UUID describes BLE profile used by the particular device, but it is not unique for each and every device (it will be the same for all BLE devices using e.g LBS). Regarding the log output I added text description of the profile in brackets, so user would not have to look for the meaning of hex number of UUID.

Reply:
ok, thank you.